test/utils.ts

```diff
@@ -39,6 +41,16 @@ export function makeCompilationEnvironment(options: Record<string, any>): Compil
     return new CompilationEnvironment(compilerProps, compilationQueue, options.doCache);
 }
 
+export function makeFakeCompilerInfo(props: Partial<CompilerInfo>): CompilerInfo {
```

Comment:
This isn't super elegant but it gets the job done. There's another `makeFakeCompilerInfo` in other parts of the code that should be merged with this at some point.